pkg/generate/app/cmd/resolve.go

```diff
@@ -502,6 +503,11 @@ func AddMissingComponentsToRefBuilder(
 				errs = append(errs, fmt.Errorf("the Dockerfile in the repository %q has no FROM instruction", info.Path))
 				continue
 			}
+			err := exposedPortsAreNumeric(dockerfileutil.LastExposedPorts(node))
+			if err != nil {
+				errs = append(errs, fmt.Errorf("the Dockerfile has an invalid EXPOSE instruction: %v", err))
+				continue
+			}
```

Comment:
i'm wondering if this is the right place for this check.  is this also going to cause "oc new-build" to fail w/ these dockerfiles?  because it probably shouldn't.

it should only be an error in cases where we're trying to generate a service(which is why we need to know what ports the dockerfile exposes).